sdk/python/pvsite_datamodel/schema.py

```diff
@@ -10,6 +10,8 @@
 class InputForecastValuesSchema(pa.SchemaModel):
     """Schema defining the makeup of the input forecast values dataframe."""
 
+    # TODO: utilise this
+
     target_datetime_utc: Series[DateTime] = pa.Field()
```

Comment:
feel free to change how you want

Reply:
Well, @simlmx and I were wondering about a potential change from 
 - having a dataframe as an argument to some of the functions and having another library to validate that

to
 - making the signature of the functions take iterables of the df column types.
 
This would be more strongly typed, be clearer as to what the function was requiring from the caller, and reduce the number of dependencies on the package. 

I don't think there is anything in the function bodies' use of the dataframe that couldn't be done with standard library functions. 

What are your thoughts on this?

Reply:
cool, go for what @simlmx did

Reply:
my only thought was https://pandera.readthedocs.io/en/stable/

Reply:
Do you reckon passing dataframes and using padera is preferable to passing standard python types?

Reply:
I would prefer a dataframe here. 

Normally I prefer pytho types, but when this is coming from a ML module, the outcome is often easier to be a dataframe. I see this as a bridge from the ML to Software world of things

Reply:
Fair enough!